pkg/controllers/flannelmigration/ipam_migrator.go

```diff
@@ -107,8 +107,8 @@ func (m ipamMigrator) InitialiseIPPoolAndFelixConfig() error {
 }
 
 // Create Calico IPAM blocks for a Kubernetes node.
-func (m ipamMigrator) SetupCalicoIPAMForNode(node *v1.Node) error {
-	log.Infof("Start setting up Calico IPAM for node %s.", node.Name)
+func (m ipamMigrator) SetupCalicoIPAMForNode(node *v1.Node, index int) error {
```

Comment:
Is passing the index just for logging purposes?

Reply:
Yes. I got an FV test case to check node ordering by capturing `node xxx[index 2]` for example.